### PR TITLE
feat: add Trtllm wideep moe collector

### PR DESCRIPTION
**Pull request by liyuanzhe1991**
#### Overview:

Add a dedicated TensorRT-LLM WideEP MoE compute-only collector that benchmarks MoE computation performance excluding All-to-All communication. This collector supports three EPLB modes and uses accurate simulation mode with DP-split and rank0 filtering.

#### Details:

**New file: `collector/trtllm/collect_wideep_moe_compute.py`**
- `WideEPMoEComputeSimulator`: Simulates WideEPMoE computation on a single GPU using the same kernel selection logic (MoEOpSelector) as TensorRT-LLM's WideEPMoE
- Supports three EPLB configurations:
  - EPLB OFF: `num_slots = num_experts` (standard routing)
  - EPLB ON (baseline): `num_slots = num_experts` (load-balanced routing)
  - EPLB ON (redundant): `num_slots = 288` (redundant expert)
- Accurate WideEP simulation mode: router uses `dp_num_tokens = num_tokens / ep_size` (single DP rank), MoE computes only tokens routed to EP rank 0
- Scope: DeepSeek-V3 model only, tp=1, ep>1, power_law distribution only

**Modified: `collector/helper.py`**
- Added `power_law_logits_v3()`: generates power-law distributed router logits with EPLB slot assignment support
- Added helper functions for greedy expert assignment and EPLB simulation

**Modified: `collector/collect.py`**
- Registered `moe_eplb` and `trtllm_moe_wideep` collection types
- Added corresponding entries to the CLI ops list

#### Where should the reviewer start?

- `collector/trtllm/collect_wideep_moe_compute.py` — the main new collector, start with `WideEPMoEComputeSimulator` class and `run_wideep_moe_compute()` function
- `collector/helper.py` — the `power_law_logits_v3()` function for EPLB-aware logit generation

#### Related Issues:

- Relates to #320 (TrtLLM WideEP pipeline support)

### CODE REVIEW

File: collector/trtllm/collect_wideep_moe_compute.py [hunk 1]
```diff
@@ -0,0 +1,1043 @@
...
+# =============================================================================
+# WideEPMoE Compute Simulator - simulates WideEPMoE computation on single GPU
+# =============================================================================
+class WideEPMoEComputeSimulator(nn.Module):
+    """
+    Simulates WideEPMoE's computation path on SINGLE GPU.
+
+    Uses the SAME kernel selection logic as WideEPMoE (MoEOpSelector.select_op).
+
+    模拟 EP size > 1 场景:
```
Comment (tianhaox) [inline collector/trtllm/collect_wideep_moe_compute.py]: let's write comment in english

File: collector/collect.py [hunk 2]
```diff
@@ -675,6 +674,26 @@ def collect_trtllm(num_processes: int, ops: list[str] | None = None):
             if v.startswith(("1.1.0", "1.2.0", "1.3.0"))
             else None,
         },
+        # MOE with EPLB (Expert Parallel Load Balancer) collection
+        {
```
Comment (tianhaox) [inline collector/collect.py:687]: do you think eplb can be part of wideep?

Comment (liyuanzhe1991) [reply collector/collect.py:687]: Yes, in fact, EPLB here is just a post-processing step for the power law – it’s simply a plugin for the MoE collector. This standalone collector was largely created because we revised the previous collection mode: we separated each individual OP within the module and assigned to each OP an input that is as close to real-world scenarios as possible.
